docs/_documentation/fundamentals/data-binding.md

```diff
@@ -439,7 +439,83 @@ where `$ViewObject$` is the view target for binding.
 
 where `$ViewProperty$` is the property on the view for binding.
 
-If `For` is not provided, then the default view property is used - e.g. for a `UILabel` the default is `Text`
+If `For` is not provided, then the default view property is used.
+
+**Android**
```

Comment:
These lists are placed without much context in this document, please back it up with some description.